dll/win32/uxtheme/CMakeLists.txt

```
@@ -1,7 +1,12 @@
 
 include_directories(${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine)
 add_definitions(-D__WINESRC__ -D__ROS_LONG64__)
-spec2def(uxtheme.dll uxtheme.spec ADD_IMPORTLIB)
+
+if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
+    spec2def(uxtheme.dll uxtheme_nt6plus.spec ADD_IMPORTLIB)
```

`vista` is usually used, instead of `nt6plus`.

```suggestion
    spec2def(uxtheme.dll uxtheme_vista.spec ADD_IMPORTLIB)
```

---

I feel like this naming convention is going to come back to bite us - `_vista` vs `_vistaplus` is the linguistic equivalent of `== vista` vs `>= vista`, after all.

That said...it *is* convention, so I'll change the name shortly.

---

For the record I would also prefer to use the term "nt6" instead of "vista"
But it's not a big deal and it's the current convention used so...
¯\_(ツ)_/¯

---

"nt6" is still "== nt6" though...

---

It's a long story, use "_vista" is okay, that means "NT6+", not NT6 or Vista only.

---

Okay, I've renamed the `uxtheme_nt6plus.spec` to `uxtheme_vista.spec`.

I still think it's a mistake to pretend people will magically somehow understand that when we use terminology that linguistically means "== Vista", we actually mean ">= Vista" (and likewise with saying "== NT 6.0" but meaning ">= NT 6.0"), but clearly that's a discussion to be had...elsewhere.